<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Breakout</class>
 <widget class="QMainWindow" name="Breakout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breakout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelCam">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PlaceHolder Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MyGLWidget" name="openGLWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string>Button Placeholder</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="previousLevel">
          <property name="text">
           <string>Previous level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextLevel">
          <property name="text">
           <string>Next level</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decreasePuckSize">
          <property name="text">
           <string>Decrease puck size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="increasePuckSize">
          <property name="text">
           <string>Increase puck size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>released()</signal>
   <receiver>Breakout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>612</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
